mitodl/mit-open
# Remove some obsolete fields from ContentFile

# What are the relevant tickets?
Closes #288 

# Description (What does it do?)
- Removes some obsolete plone-related fields from the `ContentFile` model
- Fixes a flaky test



# How can this be tested?
- Run `./manage.py recreate_index --all`
- Copy RC .env settings for `OCW_*`
- Import an OCW course or two, there should be no errors.
```
./manage.py backpopulate_ocw_data --overwrite --course-name 6-001-structure-and-interpretation-of-computer-programs-spring-2005
./manage.py backpopulate_ocw_data --overwrite  --course-name 11-014j-american-urban-history-ii-fall-2011
```
